src/main/java/org/apache/commons/codec/binary/StringUtils.java

```diff
@@ -269,6 +269,25 @@ public static ByteBuffer getByteBufferUtf8(final String string) {
         return getBytes(string, StandardCharsets.UTF_8);
     }
 
+    /**
+     * <p>Checks if a CharSequence is empty ("") or null.</p>
+     *
+     * <pre>
+     * StringUtils.isEmpty(null)      = true
+     * StringUtils.isEmpty("")        = true
+     * StringUtils.isEmpty(" ")       = false
+     * StringUtils.isEmpty("bob")     = false
+     * StringUtils.isEmpty("  bob  ") = false
+     * </pre>
+     *
+     * @param cs  the CharSequence to check, may be null
+     * @return {@code true} if the CharSequence is empty or null
+     * @since 1.16
+     */
+    public static boolean isEmpty(final CharSequence cs) {
```

Comment:
Make this package-private and move it somewhere in the language package since it is the only place where it seems to be called, which will help as noted in my other comment.

Reply:
Hi @arturobernalg ping?

Reply:
changed